Tests/FlowMagic-SDKTests/FlowMagic_SDKTests.swift

```diff
@@ -1,11 +1,11 @@
 import XCTest
-@testable import FlowMagicSDK
+@testable import FlowMagic_SDK
 
 final class FlowMagicSDKTests: XCTestCase {
     func testExample() throws {
         // This is an example of a functional test case.
         // Use XCTAssert and related functions to verify your tests produce the correct
         // results.
-        XCTAssertEqual(FlowMagic_SDK().text, "Hello, World!")
+//        XCTAssertEqual(FlowMagic_SDK().text, "Hello, World!")
```

Comment:
A little code cemetery.  You don't want to ship commented-out code to production.  Instead, please remove the entire `testExample` function.

Reply:
The commented code is removed.